src/lib/interp.rs

```diff
@@ -126,9 +133,40 @@ impl VM {
                 }
                 Instr::LOAD_GLOBAL(ref name) => panic!("NotYetImplemented"),
                 Instr::STORE_GLOBAL(ref name) => panic!("NotYetImplemented"),
-                Instr::NEW_OBJECT(ref class_name) => panic!("NotYetImplemented"),
-                Instr::LOAD_FIELD(ref field_name) => panic!("NotYetImplemented"),
-                Instr::STORE_FIELD(ref field_name) => panic!("NotYetImplemented"),
+                Instr::NEW_OBJECT(ref class_name) => {
+                    let obj = Object::new(class_name.to_string());
```

Comment:
Why do you want objects to have a class name as a string? Especially as this is going to heap allocate.

Reply:
Considering this is not used at the moment, I will remove this, but in future, I should probably refer to an object's class using a unique integer.